regression/regression_test_enkf.sh

```diff
@@ -282,9 +282,13 @@ nmem=10
 imem=1
 while [[ $imem -le $nmem ]]; do
    member="_mem"`printf %03i $imem`
-   if ! cmp -s sanl$member.${exp1} sanl$member.${exp2} 
+   ncdump incr$member.${exp1} > incr$member.${exp1}.out
+   ncdump incr$member.${exp2} > incr$member.${exp2}.out
+   if ! diff incr$member.${exp1}.out incr$member.${exp2}.out
 then
-   echo 'sanl'$member'.'${exp1}' sanl'$member'.'${exp2}' are NOT identical'
+   echo 'incr'$member'.'${exp1}' incr'$member'.'${exp2}' are NOT identical'
```

Comment:
Can you adjust indentations for this if block so it is easy to read?

Reply:
Good catch.   Indentation adjusted at 74cad42e.

Reply:
Fix syntax error at b6c56f2e.